src/main.ts

```diff
@@ -7,7 +7,7 @@ export default class AdvancedCloseTab extends Plugin {
 
   addCommands() {
     this.addCommand({
-      id: 'advanced-close-tab-close-current-tab',
+      id: 'close-current-tab',
```

Comment:
This would become a breaking change, so it would be preferable to make the change in a separate pull request.
For instance, re-registration of Hot Keys will be necessary. That was the reason I was avoiding changing the existing command IDs. Sorry for the lack of explanation.
Since this plugin is not registered as a community plugin, I think the users will understand the changes...

Reply:
I'm sorry for the trouble, but could you possibly revert the changes here?

Reply:
✗ left → ○ leave 😓

Reply:
> For the new command ID, can I delete the plugin name prefix? Or should I left the prefix to keep consistency?

Yes, for the new command, we would like to go without adding the prefix.

Reply:
I understand. I've modified it 🤲